nomenclature/definitions/variable/technology/power-plant.yaml

```
@@ -3,638 +3,153 @@
-   unit: MW
-
-Minimum Active power|Electricity|Hydro|Pumped Storage:
-   description: Minimum active power of one  "Pumped Storage" unit
-   unit: MW
-
+# specific variables for pumped storage hydro power plants
 Maximum Storage|Electricity|Hydro|Pumped Storage:
-   description: Maximum volume of a Pumped Storage expressed in energy
+   description: Maximum volume of a pumped storage system
+     expressed in terms of energy
    unit: MWh
 
 Minimum Storage|Electricity|Hydro|Pumped Storage:
-   description: Minimum volume of a Pumped Storage expressed in energy
-   unit: MWh
+  description: Minimum volume of a pumped storage system
+    expressed in terms of energy
+  unit: MWh
 
 Pumping Efficiency|Electricity|Hydro|Pumped Storage:
-   description: efficiency of pumping for a Pumped Storage
-   unit: "%"
-
-Inertia|Electricity|Hydro|Pumped Storage:
-   description: inertia provided to the system by a Pumped Storage unit
-   unit: second
-
-Rate Frequency Containment Reserve|Electricity|Hydro|Pumped Storage:
-   description: Percentage of the Maximum Active Power that can be devoted to Frequency Containment Reserve
-                for a Pumped Storage unit
+   description: Pumping efficiency for a reservoir power plant
    unit: "%"
 
-Rate Automatic Frequency Restoration Reserve|Electricity|Hydro|Reservoir:
-   description: Percentage of the Maximum Active Power that can be devoted to Automatic Frequency Restoration Reserve
-                for a Pumped Storage unit
-   unit: "%"
-
-# description of simple hydrosystems -  run of river
-# this describes systems composed of one run of river plant (without storage)
-Number of Units|Electricity|Hydro|Run of River:
-   description: Number of units of kind "Run of River"
-   unit:
-
-Maximum Active power|Electricity|Hydro|Run of River:
-   description: Maximum active power of one  "Run of River" unit
-   unit: MW
-
-LoadFactor|Electricity|Hydro|Run of River|Profile:
-   description: Timeserie of hourly loadfactors for  "Run of River"
-   unit: "%"
-
-Minimum Active power|Electricity|Hydro|Run of River:
-   description: Minimum active power of one  "Run of River" unit
-   unit: MW
-
-Inertia|Electricity|Hydro|Reservoir:
-   description: inertia provided to the system by a typical "simple Hydro reservoir " unit
-   unit: second
-
-Rate Frequency Containment Reserve|Electricity|Hydro|Reservoir:
-   description: Percentage of the Maximum Active Power that can be devoted to Frequency Containment Reserve
-                for a typical "simple Hydro reservoir " unit
-   unit: "%"
-
-Rate Automatic Frequency Restoration Reserve|Electricity|Hydro|Reservoir:
-   description: Percentage of the Maximum Active Power that can be devoted to Automatic Frequency Restoration Reserve
-                for a typical "simple Hydro reservoir " unit
+# specific variables for run-of-river hydro power plants
+Load Factor|Electricity|Hydro|Run of River|Profile:
+   description: Load of hourly loadfactors for "Run of River"
```

As the name of the variable was changed, I would also change the description
```suggestion
   description: Load of hourly load factors for "Run of River"
```